actions/storage.ts

```diff
@@ -1,11 +1,13 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Run time code file, but all changes are at type level, so they do not impact actual execution.